extension/src/time_vector/pipeline/map.rs

```diff
@@ -152,11 +102,11 @@ pub fn check_user_function_type(function: pg_sys::regproc) {
         error!("invalid number of mapping function arguments, expected fn(timevector) RETURNS timevector")
     }
 
-    if unsafe { *argtypes } != *crate::time_series::TIMEVECTOR_OID {
+    if unsafe { *argtypes } != *crate::time_vector::TIMEVECTOR_OID {
```

Comment:
Can we `assert!(!nargs.is_null())` before dereferencing? I'd always NULL-check in C too. I prefer surprising "impossible NULL at file:NNN" in my logs to surprise segfaults :)

Reply:
I assume you mean `argtypes` rather than `nargs` here.  Added the assert.